rhel8/templates/csv/packages_installed.csv

```diff
@@ -9,6 +9,7 @@ libreswan
 ntp
```

Comment:
I have kept `ntp` here because rule `service_chronyd_or_ntpd_enabled` relies on check for `package_ntp_installed`.
`chronyd_or_ntpd` rules need a `chronyd` only rule version befor ntp can be removed from this CSV.

Reply:
That could be achieved by JInja macros in the rule, OVALs and remediations.

Reply:
I could filter out checks for `ntp` and make it work, but the rule would still be called `chronyd_or_ntpd`, which would be confusing...

Reply:
We can rename the rule to `clock_synchronization_enabled`. But I agree with keeping `ntp` here, it's easier.

Reply:
There is already a ticket to break out these rules which should be separate rules and not rules joined by jinja.